integrations/gsheets/src/index.ts

```diff
@@ -1,8 +1,20 @@
 import actions from './actions'
 import * as bp from '.botpress'
+import { getClient } from './client'
 
 export default new bp.Integration({
-  register: async () => {},
+  register: async (props) => {
+    props.logger.forBot().info('Registering Gsheets integration')
+    try {
+      const gsheetsClient = getClient(props.ctx.configuration)
+      const { properties, sheets } = await gsheetsClient.getSpreadsheet('')
+      const summary = JSON.stringify({ properties, sheets }).substring(0, 200)
+      props.logger.forBot().info(`Successfully connected to Gsheets: ${summary}`)
```

Comment:
I don't think the summary is going to be useful in the logs
```suggestion
      props.logger.forBot().info(`Successfully connected to Gsheets`)
```